cmake/modules/SwiftSupport.cmake

```
@@ -40,7 +40,7 @@ function(get_swift_host_arch result_var_name)
     set("${result_var_name}" "armv7" PARENT_SCOPE)
   elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "armv7-a")
     set("${result_var_name}" "armv7" PARENT_SCOPE)
-  elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "AMD64")
+	elseif("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "AMD64|amd64")
```

Nit: Indentation is slightly off
```suggestion
  elseif("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "AMD64|amd64")
```